etc/config/fortran.amazon.properties

```diff
@@ -118,6 +118,13 @@ compiler.farmg730.name=ARM gfortran 7.3
 compiler.farmg820.exe=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gfortran
 compiler.farmg820.name=ARM gfortran 8.2
 
+###############################
+# LLVM Flang for X86
+group.clang_llvmflang.compilers=flang
```

Comment:
Can you rename the flag compiler id to something like `flangtrunk` so we are ready for the future in the case that more versions are added? Thanks! :)

Reply:
@RabsRincon do you mean `s/flang/flangtrunk` ?

Reply:
I am assuming the same. I have pushed PR.